### PR TITLE
Feature/style fixes

### PR DESCRIPTION
**Pull request by JackCrescend**
Adjusted image scaling to prevent images from being stretched beyond their original resolution.

Added gura theme colors, and hard-coded projects/3 to display guras colors. Hard-coding gets around the issue of color flash while project is still loading & page doesn't know which streamer page belongs to.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -85,7 +74,7 @@ export default function Home() {
 				<div className="my-16 w-full flex flex-col items-center">
 					<div className="max-w-4xl w-full mx-4">
 						<div>
-							<TextHeader text="Featured projects" />
+							<TextHeader text="Ongoing projects" />
```
Comment (GoldElysium) [inline pages/index.tsx:77]: Featured is not supposed to show all ongoing, it's supposed to show three selected by the admin. This hasn't been added yet, so it used the first three in the db. But, we can leave it like this I guess.

File: pages/index.tsx [hunk 2]
```diff
@@ -48,8 +48,10 @@ export default function Home() {
 			.then((response) => response.json())
 			.then((data) => {
 				let projectHtml = [];
-				if (data.length <= 3) {
-					projectHtml = data.map((project: IProject) => (
+
+				projectHtml = data
+					.filter((project: IProject) => project.status === 'ongoing')
+					.map((project: IProject) => (
```
Comment (GoldElysium) [inline pages/index.tsx:54]: Put a max limit on this please, otherwise this is just another projects page.

**Review by GoldElysium — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 2 above)
Comment (JackCrescend) [reply pages/index.tsx:55]: All right I put it back to 3, it's basically the same as before but now filtering out past projects.

**Review by GoldElysium — APPROVED** [review]

**Comment by GoldElysium** [comment]
> Hard-coding gets around the issue of color flash while project is still loading & page doesn't know which streamer page belongs to.
This could also be prevented by using server side rendering. This will also decrease the cumulative layout shift.

**Pull request merged**